### DISCUSSION TITLE
favicon.ico is not shown next 14.0.4

### DISCUSSION THREAD

**Discussion by FerelUltra** [vercel/next.js#60190] [App Router]
I have bumped into favicon issue. Have checked a lot of problem resolves, but then realized even create-next-app has no favicon by default (even though next must read favicon.ico in app by default).  So who have stucked too?

**Comment by ananuness** [comment]
Hii! I checked running `npx create-next-app@14.0.4 and that was generated:

![image](https://github.com/vercel/next.js/assets/51466624/cac5e234-0a96-41f1-9997-d664ffddd7bd)

As you can see, the `favicon.ico` is present and here is working fine. Have you tried to add any `.ico` image to app folder?
